src/cljs/main/broadfcui/page/profile.cljs

```diff
@@ -26,6 +26,14 @@
         (let [loc (.-location js/window)]
           (str (.-protocol loc) "//" (.-host loc) "/#profile/nih-username-token={token}")))))
 
+(defn get-fence-link-href []
```

Comment:
all of the todos are related to this link. need to read the client ID and root URL from config. also need to get the redirect URL approved for `http` instead of just `https` and switch URL to `/#fence-callback` instead of `/fence-callback`

lastly there is a state parameter that must be present for this link to succeed.